domain/domain.go

```diff
@@ -132,6 +132,7 @@ type Domain struct {
 	indexUsageSyncLease   time.Duration
 	dumpFileGcChecker     *dumpFileGcChecker
 	planReplayerHandle    *planReplayerHandle
+	extractTaskHandle     *extractHandle
```

Comment:
Can we just merge this handler with `planReplayerHandle` to let it simpler? 
Then we can implement all features related to `plan information dumping` together.